starknet-foundry/crates/cast/src/starknet_commands/account/deploy.rs

```diff
@@ -1,5 +1,115 @@
+use crate::helpers::constants::OZ_CLASS_HASH;
+use anyhow::{anyhow, bail, Result};
+use camino::Utf8PathBuf;
+use cast::{get_network, handle_rpc_error, parse_number, print_formatted, wait_for_tx};
 use clap::Args;
+use starknet::accounts::AccountFactoryError;
+use starknet::accounts::{AccountFactory, OpenZeppelinAccountFactory};
+use starknet::core::types::FieldElement;
+use starknet::providers::jsonrpc::HttpTransport;
+use starknet::providers::{JsonRpcClient, Provider};
+use starknet::signers::{LocalWallet, SigningKey};
 
 #[derive(Args, Debug)]
 #[command(about = "Deploy an account to the Starknet")]
-pub struct Deploy {}
+pub struct Deploy {
+    /// Name of the account to be deployed
+    #[clap(short, long)]
+    pub name: String,
+
+    /// Max fee for the transaction
+    #[clap(short, long)]
+    pub max_fee: FieldElement,
+}
+
+pub async fn deploy(
+    provider: &JsonRpcClient<HttpTransport>,
+    path: Utf8PathBuf,
+    name: String,
+    network: &str,
+    max_fee: FieldElement,
+) -> Result<FieldElement> {
+    let network = get_network(network)?.get_value();
+
+    let contents = std::fs::read_to_string(path.clone()).expect("Couldn't read accounts file");
+    let mut items: serde_json::Value =
+        serde_json::from_str(&contents).expect("failed to parse json file");
```

Comment:
```suggestion
        serde_json::from_str(&contents).expect("Failed to parse json file");
```
nitpick, sorry I didn't catch this previously

Reply:
Maybe we could add a file name in errors as well? `Failed to parse json file` does not tell much